admin/app/helpers/workarea/admin/content_helper.rb

```diff
@@ -6,6 +6,15 @@ def current_system_page_content_for(name)
         @tmp[name.to_s.systemize] ||= Content.for(name.to_s.systemize)
       end
 
+      def edit_layout_footer_area_path
+        content = Admin::ContentViewModel.wrap(
+          current_system_page_content_for(:layout)
+        )
+
+        area_id = content.areas.find { |a| a =~ /footer/i }
+        edit_content_path(content, area_id: area_id || content.areas.first)
```

Comment:
I think it would be better to just not render the link if we can't find an area explicitly named with "footer". Like I could see this logic end up linking to "header promo" area. That would be a weird UX then.

Reply:
Yep, that makes more sense.